### DISCUSSION TITLE
ShellAllowlistOpen for `vscode://` protocol

### DISCUSSION THREAD

**Discussion by grikomsn** [tauri-apps/tauri#9350] [Q&A]
How does one handle opening `vscode://` protocols which should open VS code on click? Use case is debugging a Tauri + Vite + React app with [LocatorJS](https://github.com/infi-pc/locatorjs) which should open VS Code when clicking a component.

**Comment by FabianLars** [comment]
Basing it on the default from https://tauri.app/v1/api/config#shellallowlistopen it'd be something like this i guess
```
"open": "^((mailto:\w+)|(tel:\w+)|(https?://\w+)|(vscode://\w+)).+"
```
